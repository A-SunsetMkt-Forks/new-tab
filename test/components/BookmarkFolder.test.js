'use strict';

const BookmarkFolder = require('../../src/components/BookmarkFolder.html');

const _node = {};

describe('BookmarkFolder component', () => {
  it('renders correctly', () => {
    const target = document.createElement('div');
    new BookmarkFolder({
      target,
      data: { _node },
    });
    expect(target.innerHTML).toMatchSnapshot();
  });
});
