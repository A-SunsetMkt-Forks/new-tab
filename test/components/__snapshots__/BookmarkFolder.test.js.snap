// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BookmarkFolder component renders correctly 1`] = `
"<div class=\\"item folder\\" title=\\"undefined\\">undefined

  <div class=\\"caret\\">â–¸</div>

  </div>"
`;
